templates/frontend/pages/submissions.tpl

```diff
@@ -83,6 +58,21 @@
 	</div>
 	{/if}
 
+	{foreach from=$sections item="section"}
+		{if $section->getLocalizedPolicy()}
```

Comment:
This condition prevents a section from being displayed on the about submissions page when no section policy was found. That's why i originally handled policy and section ids separately. We should display the section nontheless to be consistent and provide the direct submission link. What do you say?

Reply:
I am not sure I agree. The main submission link at the top will be the primary submission route for authors, and they can select any section they want. My understanding is that we only need to display Section Policies here so that authors can be informed of unique requirements for special sections before making the submission.

Reply:
It sounds like you have a highly customized setup built around the Section Policies. I don't think this will be common for our users. In most cases, the Section Policy will say a few things like "Articles are XXXX words and peer reviewed and we accept on these topics and our usual publishing schedule is XXXX." "Reviews are XXXX words and not peer reviewed and ..."

If you'd like, I'd be happy to offer some help to get your submissions page looking the way that you'd like with a custom theme.  Are you already using a custom theme? If you're using an out-of-the-box theme, I can whip up a quick child theme that overwrites the submissions template that suits your original design approach.

Reply:
Yes, in a custom theme you'll need to overwrite the `frontend/pages/submissions.tpl` template file. With a child theme, it should be just that one template file. 

We could split off the `foreach` loop into a separate template file. But please understand that there are hundreds, maybe thousands, of journals that run customized template files. If we split every template file into smaller and smaller files to suit their customizations, it will become increasingly difficult for a new developer to locate the code that they want to modify.

It's not always easy, but we try to balance modularity with template simplicity, and for the frontend we tend to err on the side of template simplicity because that's where most new developers will first encounter our platform.

I'll extend the invitation again: if you let me know what theme you're using, I'll whip up a child theme plugin to get you 90% of the way there.

Reply:
Ok, then i have to stick with a customized template that needs merging on each update.

Reply:
With a child theme, you would not need to merge on each update. You could simply check the [history of the overwritten template](https://github.com/pkp/pkp-lib/commits/master/templates/frontend/pages/submissions.tpl) and see if any changes were made since the last update.

It looks like the last substantive change to `submissions.tpl` was in Dec 2017. It's not a file that sees a lot of turnover.